.github/workflows/vm.yaml

```diff
@@ -1,4 +1,4 @@
-name: VM
+name: CGroupv2
```

Comment:
can you avoid this in this PR?

Reply:
Yeah, I'll switch it back. Ben had commented on some confusion based on that naming at the same time I had the file open, so I quick threw that in. But it's a different concern from the rest of the change.

Reply:
I think we should do this, it's non-obvious.

In the future we'll rewrite these and be testing with cgroup v1 in a vagrant job while the rest of CI is on v2.

Reply:
How about this... the overall workflow is named `VM` and the single job under the workflow is also named `VM`. This is why we end up with the displayed name of `VM/VM`. I can keep the workflow named `VM` to denote these are jobs that run with a VM, then rename the job itself to be `CGroupv2`. That would result in the running jobs (multiple because it's a matrix job for each combination) being named `VM/CGroupv2`.

Then if we choose to add other jobs that need to be run within a VM, we could have something like:

- `VM/CGroupv2`
- `VM/CGroupv1`
- `VM/Windows`
- `VM/foo`

etc.

If we have some consensus if that makes sense, I can quick update again.

Reply:
Especially since it looks like I'll have to do an update anyway because I missed something. :D

Reply:
I remember now, I just didn't want to call this Fedora https://github.com/kubernetes-sigs/kind/pull/3485 so it move to something generic ... basically because this can be a magnet for distro validation

+1 on the VM/faeture format